packages/node-opcua-end2end-test/test/end_to_end/test_e2e_issue896_execute_method_access_control.js

```diff
@@ -0,0 +1,89 @@
+"use Strict";
```

Comment:
use strict

Reply:
migrated js => ts

Reply:
@erossignon Who gonna "Resolve conversation"? Is there any convention?
Should it be always the initiator of the convrsation or can I hit the button, if I am convinced that it is fixed?